consensus/service/src/api/client_api_service.rs

```diff
@@ -188,6 +192,7 @@ impl ConsensusClientApi for ClientApiService {
         result = result.and_then(|mut response| {
             let num_blocks = self.ledger.num_blocks().map_err(ConsensusGrpcError::from)?;
             response.set_block_count(num_blocks);
+            response.set_block_version(*self.config.block_version);
```

Comment:
If this is approved @christian-oudard & FS should be notified to update their protos responses to include it as well